aiocouchdb/tests/test_feeds.py

```diff
@@ -108,19 +108,16 @@ def test_buffer_workflow(self):
         self.assertTrue(feed.is_active())
 
         result = yield from feed.next()
-        self.assertEqual(feed._queue.qsize(), buf_size)
```

Comment:
It was testing internal state of the object; `asyncio.Queue` is well tested by it's own library and its correctness shouldn't be a concern here, should it?